unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatEntry.cs

```diff
@@ -262,6 +335,20 @@ void ProcessHoverPanelTimer()
         }
     }
 
+    void ProcessHoverGotoPanelTimer()
+    {
+        if (hoverGotoPanelTimer <= 0f || !isOverCoordinates)
+            return;
+
+        hoverGotoPanelTimer -= Time.deltaTime;
+        if (hoverGotoPanelTimer <= 0f)
```

Comment:
cannot we substract `Time.deltaTime`  earlier and have only one check?

Reply:
i don't think so, this is the same behavior as the old ProcessHoverPanel (for the timestamp of the message) the division in 2 checks is made to work as a flag for a 1 time execution for the condition